conda/cli/main_config.py

```diff
@@ -676,6 +693,11 @@ def execute_config(args, parser):
             rc_path = user_rc_path
     elif args.file:
         rc_path = args.file
+    elif args.prefix or args.name:
+        if is_conda_environment(context.target_prefix):
```

Comment:
Not sure if this is too strict? Should a user be able to specify a `prefix` or `environment` that does not exist?